test/safe_pg_migrations_test.rb

```diff
@@ -133,32 +133,34 @@ def up
         end
       end.new
 
-    execute_calls = nil
-    write_calls =
-      record_calls(@migration, :write) do
-        execute_calls = record_calls(@connection, :execute) { run_migration }
-      end
-    assert_calls [
-      # The column is added without any default.
-      'ALTER TABLE "users" ADD "admin" boolean',
+    SafePgMigrations.stub(:get_pg_version_num, 96_000) do
```

Comment:
why this stub?

Reply:
Otherwise, PG uses your local version. As I had PG 11 installed, the test was not working for me. This fixes it